.travis.yml

```diff
@@ -36,9 +44,23 @@ before_script:
   - sh -c 'if [ "$SYMFONY_VERSION" != "" ]; then composer require -n --no-update symfony/symfony=$SYMFONY_VERSION; fi;'
   - composer install --prefer-source
 
-  - sudo apt-get update
-  - sudo apt-get install -y --force-yes apache2 libapache2-mod-php5
-  - sudo sed -i -e "s,/var/www,$(pwd)/Tests/Resources/public,g" /etc/apache2/sites-available/default
-  - sudo /etc/init.d/apache2 restart
+  - sudo chmod -R 0777 Tests/logs
+  - sudo apt-get update > /dev/null
+  - sudo apt-get install -y --force-yes apache2 libapache2-mod-fastcgi > /dev/null
+  # enable php-fpm
+  - sudo cp ~/.phpenv/versions/$(phpenv version-name)/etc/php-fpm.conf.default ~/.phpenv/versions/$(phpenv version-name)/etc/php-fpm.conf
+  - sudo a2enmod rewrite actions fastcgi alias
+  - echo "cgi.fix_pathinfo = 1" >> ~/.phpenv/versions/$(phpenv version-name)/etc/php.ini
+  - ~/.phpenv/versions/$(phpenv version-name)/sbin/php-fpm
```

Comment:
Instead of using Apache and php-fpm, which require sudo access and changing permissions on logs, you might try to rely on the PHP builtin webserver, using the same trick than in driver repos to run it for PHP 5.3 or HHVM (see in GoutteDriver for instance)